### PR TITLE
Inline usages of configurations.load

### PR DESCRIPTION
**Pull request by ynurmahomed**
* Use only one service for configurations
* Load patient configurations inside `appService.initApp`

### CODE REVIEW

**Comment by ynurmahomed** [comment]
[Inline usages of `configurations.load`](https://trello.com/c/zpfMwOcU/8-inline-usages-of-configurationsload)

File: app/home/index.html [hunk 1]
```diff
@@ -98,9 +98,9 @@
   <script src="../common/domain/services/encounterService.js"></script>
   <script src="../common/domain/services/consultationService.js"></script>
 
-
-  <script src="../common/config/init.js"></script>
-  <script src="../common/config/services/configurations.js"></script>
+  <script src="../common/patient/constants.js"></script>
+  <script src="../common/patient/models/patientConfig.js"></script>
+  <script src="../common/patient/mappers/patientAttributeTypeMapper.js"></script>
```
Comment (drryanjames) [inline app/home/index.html]: Are you alphabetizing these files?

File: app/patient-details/components/patientDetails.js [hunk 2]
```diff
@@ -44,6 +41,11 @@
         .catch(function () {
           notifier.error(translateFilter('COMMON_MESSAGE_ERROR_ACTION'));
         });
+
+      configurationService.getAddressLevels()
+        .then(function (addressLevels) {
+          vm.addressLevels = addressLevels;
```
Comment (drryanjames) [inline app/patient-details/components/patientDetails.js]: In some of your other check ins there was was .routes.js file that resolved vm properties. Will this file eventually have a .routes file and will this eventually move into the resolve object?

**Comment by ynurmahomed** [comment]
@edrisse can you  review, this affects create/edit/ patient and patient details.

File: app/home/index.html (hunk 1 above)
Comment (ynurmahomed) [reply app/home/index.html]: Not alphabetizing, I think it's a coincidence.

File: app/patient-details/components/patientDetails.js (hunk 2 above)
Comment (ynurmahomed) [reply app/patient-details/components/patientDetails.js]: Yes. It's already used in some `.routes`, but the component is missing `bindings: { patient: '<' }` in its definition.

I still don't know whats the best approach for loading this data [controller-activation-promises](https://github.com/johnpapa/angular-styleguide/tree/master/a1#controller-activation-promises) vs  [route-resolve-promises](https://github.com/johnpapa/angular-styleguide/tree/master/a1#route-resolve-promises).

**Pull request merged**